stylesheets/styles.css

```diff
@@ -14,3 +14,26 @@ div {
 	margin-left: 25%;
 	margin-right: 25%;
 }
+
+button {
+	border-style: solid;
+	border-radius: 2px;
+	font-size: 14px;
+	font-weight: bold;
```

Comment:
recommend an ```outline:none``` here to prevent the browser from giving the buttons an extra highlight when clicked.